pwa/src/templates/templateParts/userForm/UserFormTemplate.tsx

```diff
@@ -79,6 +85,13 @@ export const UserFormTemplate: React.FC<UserFormTemplateProps> = ({ user }) => {
     );
   }, [getOrganization.isSuccess]);
 
+  React.useEffect(() => {
+    setValue(
+      "applications",
+      user["applications"].map((application: any) => ({ label: application.name, value: application.id })),
```

Comment:
I think we can target `["applications"]` like `.applications` instead, to be more consistent with other implementations.

Reply:
~~Creating a user breaks with this implementation, because `user` doesn't exist in that context, so `user.applications` will break.~~

~~Solution: `user?.application...`.~~

Edit: we shouldn't be setting these here altogether, see comment on dependency array.